src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.Semaphore.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        [DllImport(Interop.Libraries.Kernel32, SetLastError = true, CharSet = CharSet.Unicode)]
+        internal static extern SafeWaitHandle OpenSemaphore(uint desiredAccess, bool inheritHandle, string name);
```

Comment:
These methods had `EntryPoint` explicitly specified before. It would be nice to keep it. It makes them a bit faster to load at runtime, and it may be actually required in some cases in CoreRT/.NET Native for correctness.

(Make the same change in `Interop.Mutex.cs` as well.)